docs/vale.md

```diff
@@ -164,6 +164,23 @@ Vale can be forced to ignore sections of text using the following syntax:
 ```
 Everything in that section will be ignored. Consider using this exclusively for edge-cases where text should be used in a way that does not fit our current rule-set.
 
+
+## Ignoring raw link text
+
+Because our URL's contain product names they will trigger terms or dictionary checks. 
+If you need to ignore raw link text in URL's here are two options:
+
+Adding the `bare` class to the link: 
+
+```
+<a href="https://konnect.konghq.com" class="bare">konnect.konghq.com</a>
```

Comment:
Have you tested this with the markdown class syntax? Eg:

```
[link](/link/){:.bare}
```

Reply:
I have to test this one sec

Reply:
Unfortunately, this didn't work as I needed it to, and it highlighted that what I implemented didn't work like I thought it was working either.